tools/cluster_mgr.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import math
+import redis
+import subprocess
+import time
+
+'''
+To install: pip install -r requirements.txt
+'''
+
+
+class Node:
+    def __init__(self, port, admin_port):
+        self.port = port
+        self.admin_port = admin_port
+
+    id = ''
+
+
+def start_node(node, threads):
+    f = open(f'/tmp/dfly.cluster.node.{node.port}.log', 'w')
+    print(f'- Log file for node {node.port}: {f.name}')
+    subprocess.Popen(['../build-dbg/dragonfly', f'--port={node.port}',
+                      f'--admin_port={node.admin_port}', '--cluster_mode=yes', f'--proactor_threads={threads}',
+                      '--dbfilename=', f'--logtostderr'], stderr=f)
```

Comment:
i would wait here for 100ms and check the result of Popen to eliminate the obvious errors when the process fails on a start.

Reply:
The thing is, we call this function `n` times, but only the first of them will likely wait
Not that this is of any significance :)